cmd/limactl/volume.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"text/tabwriter"
+
+	"github.com/docker/go-units"
+	"github.com/lima-vm/lima/pkg/qemu"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newVolumeCommand() *cobra.Command {
+	var volumeCommand = &cobra.Command{
+		Use:   "volume",
+		Short: "Lima volume management",
+		Example: `  Create a volume:
+  $ limactl volume create data_volume
+
+  List existing volumes:
+  $ limactl volume ls
+
+  Delete a volume:
+  $ limactl volume delete data_volume`,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+	volumeCommand.AddCommand(
+		newVolumeCreateCommand(),
+		newVolumeListCommand(),
+		newVolumeDeleteCommand(),
+		newVolumeUnlockCommand(),
+	)
+	return volumeCommand
+}
+
+func newVolumeCreateCommand() *cobra.Command {
+	var volumeCreateCommand = &cobra.Command{
+		Use: "create VOLUME",
+		Example: `
+To create a new volume:
+$ limactl volume create VOLUME
+`,
+		Short: "Create a Lima volume",
+		Args:  cobra.ExactArgs(1),
+		RunE:  volumeCreateAction,
+	}
+	volumeCreateCommand.Flags().String("size", "2G", "configure the volume size")
+	return volumeCreateCommand
+}
+
+func volumeCreateAction(cmd *cobra.Command, args []string) error {
+	_, err := createVolume(cmd, args)
```

Comment:
Why do you need a separate `createVolume()` function that just returns an extra value that is being ignored here.
The function isn't called from anywhere else, so why isn't it inlined here instead?